Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: all docs test itest
+
+docs: .taskproc/bin/activate
+	.taskproc/bin/tox -e docs
+
+test: .taskproc/bin/activate
+	.taskproc/bin/tox
+
+itest:
+	cd tests/integration && \
+	docker-compose -f cluster/docker-compose.yaml down && \
+	docker-compose -f cluster/docker-compose.yaml pull && \
+	docker-compose -f cluster/docker-compose.yaml build && \
+	docker-compose -f cluster/docker-compose.yaml up -d zookeeper mesosmaster mesosslave && \
+	docker-compose -f cluster/docker-compose.yaml scale mesosslave=2 && \
+	docker-compose -f cluster/docker-compose.yaml run playground /src/itest.sh
+
+.taskproc/bin/activate:
```

Comment:
How about, instead of just creating a hidden virtualenv for installing tox, we create one that will function as the development environment as well, i.e. where we install all of task_processing's runtime and development dependencies into?